bitdefender/assets/logs/bitdefender.yaml

```diff
@@ -0,0 +1,16 @@
+id: bitdefender
+metric_id: bitdefender
+backend_only: false
+facets: null
+pipeline:
+  type: pipeline
+  name: Bitdefender
+  enabled: true
+  filter:
```

Comment:
Why are no facets defined? There are some opportunities to remap to standard attributes here.
Take a look at various standard facets here: https://docs.datadoghq.com/standard-attributes/?product=log+management


1. Would it make sense to map `computer_name` or `computer_id` map to host?
2. `computer_ip` -> `network.client.ip`?
3. `url` -> `http.url`?
4. What are the potential values of `status`? Is there a category remapped you can implement to map the status of the Datadog log (info, warning, error, etc) to the value

Reply:
Excellent, it looks good. I'm going to add the staging tag and see if we can get this validated in staging. Thanks for updating the pipeline.